docs/UserGuide.md

```diff
@@ -270,9 +270,8 @@ Format: `add n/TASK NAME m/MODULE d/DESCRIPTION [a/START TIME] b/DEADLINE w/WORK
 * Recurrence should be either "daily", "weekly" or "biweekly".
 
 * Enter the time you wish to complete your task in any of these formats: 
-  * yyyy-MM-dd HH:mm:ss
-  * yyyy-MM-dd HH:mm (ss set to 0)
-  * yyyy-MM-dd (HH:mm:ss set to 00:00:00)
+  * yyyy-MM-dd HH:mm
+  * yyyy-MM-dd (HH:mm set to 00:00)
```

Comment:
In my last edit to the UG, I added seconds to some examples. If we are getting rid of seconds, we need to remove them from the command examples.

Reply:
I only find one occurrence from the command example (editCommand), is that correct?
I decide to get rid of seconds because we need to update the functionality of recur and update error messages.

Reply:
Maybe just push it to the main repo and see what happens 😆 the UI may have some problems but I can test it out